src/main/resources/assets/betterquesting/lang/en_US.lang

```diff
@@ -90,6 +91,7 @@ betterquesting.tooltip.quest_logic=Quest Logic: %s
 betterquesting.tooltip.simultaneous=Simultaneous Tasks: %s
 betterquesting.tooltip.auto_claim=Auto Claim: %s
 betterquesting.tooltip.repeat=Cooldown: %s
+betterquesting.tooltip.repeat_with_edit_mode=Edit mode enabled, repeatable quests won't resets
```

Comment:
> won't resets 

The s should be removed

Reply:
@Prometheus0000 why. it refer to an issue somone reported that have edit mode on and run in the issue the quest not reseted

Reply:
Because it isn't grammatically correct the way it is? For example your sentence is wrong too, it should be

> why. it refer to an issue somone reported that have edit mode on and run in the issue the quest not reseted

Why? It refers to an issue someone reported that was caused by having edit mode on, and ran into an issue with the quest not resetting.

Having an s there is grammatically incorrect.

Reply:
Yes understood.